### PR TITLE
docs | tasks | configure-pod-container | pull-image-private

### PR DESCRIPTION
**Pull request by icheikhrouhou**
Translate to fr the following : pull-image-private-registry.md

### CODE REVIEW

**Comment by sftim** [comment]
This PR looks valid, I think it just needs reviewing
/remove-lifecycle stale

**Comment by sftim** [comment]
@kubernetes/sig-docs-fr-reviews would anyone be able to take a look?

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 1]
```diff
@@ -0,0 +1,209 @@
+---
+title: Extraction d'une image d'un registre privé
```
Comment (rekcah78) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:2]: title: Récupération d'une image d'un registre privé

Comment (rekcah78) [reply content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:2]: Je propose cette modification qui me semble plus adapté que le mot Extraction.

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+```shell
+kubectl apply -f my-private-reg-pod.yaml
+kubectl get pod private-reg
+```
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+* Pour en savoir plus sur [Secrets](/docs/concepts/configuration/secret/).
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:201]:
```suggestion
* Pour en savoir plus sur les [Secrets](/docs/concepts/configuration/secret/).
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+* Pour en savoir plus sur [en utilisant un registre privé](/docs/concepts/containers/images/#using-a-private-registry).
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:202]:
```suggestion
* Pour en savoir plus sur l'[utilisation d'un registre privé](/docs/concepts/containers/images/#using-a-private-registry).
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 4]
```diff
@@ -0,0 +1,209 @@
...
+* Pour en savoir plus sur [ajouter l' imagePullSecrets à un compte de service](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:203]:
```suggestion
* Pour en savoir plus sur l'[ajout d'un imagePullSecrets à un compte de service](/docs/tasks/configure-pod-container/configure-service-account/#add-imagepullsecrets-to-a-service-account).
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 5]
```diff
@@ -0,0 +1,209 @@
...
+wget -O my-private-reg-pod.yaml https://k8s.io/examples/pods/private-reg-pod.yaml
+```
+
+Dans le fichier `my-private-reg-pod.yaml`, remplacez `<your-private-image>` par le chemin d'accès à une image dans un registre privé tel que
+
+```none
+your.private.registry.example.com/janedoe/jdoe-private:v1
+```
+
+Pour extraire l'image du registre privé, Kubernetes a besoin des identifiants.
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:187]:
```suggestion
Pour récupérer l'image du registre privé, Kubernetes a besoin des identifiants.
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 6]
```diff
@@ -0,0 +1,209 @@
...
+echo "c3R...zE2" | base64 --decode
+```
+
+La sortie en tant que nom d'utilisateur et mot de passe concaténés avec un `:`, est similaire à ceci :
+
+```none
+janedoe:xxxxxxxxxxx
+```
+
+Remarquez que les données Secrètes contiennent le token d'autorisation similaire à votre fichier local `~/.docker/config.json`.
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:165]:
```suggestion
Remarquez que les données secrètes contiennent le token d'autorisation similaire à votre fichier local `~/.docker/config.json`.
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 7]
```diff
@@ -0,0 +1,209 @@
...
+
+* `<your-registry-server>` est votre FQDN de registre de docker privé. (https://index.docker.io/v1/ for DockerHub)
+* `<your-name>` est votre nom d'utilisateur Docker.
+* `<your-pword>` est votre mot de passe Docker.
+* `<your-email>` est votre email Docker.
+
+Vous avez réussi à définir vos identifiants Docker dans le cluster comme un secret appelé `regcred`.
+
+{{< note >}}
+Saisir des secrets sur la ligne de commande peut les conserver dans l'historique de votre shell sans protection, et ces secrets peuvent également être visibles par d'autres utilisateurs sur votre PC pendant le temps que `Kubectl` est en cours d'exécution.
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:113]:
```suggestion
Saisir des secrets sur la ligne de commande peut les conserver dans l'historique de votre shell sans protection, et ces secrets peuvent également être visibles par d'autres utilisateurs sur votre ordinateur pendant l'exécution de `kubectl`.
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 8]
```diff
@@ -0,0 +1,209 @@
...
+
+Si vous avez déjà lancé `docker login`, vous pouvez copier ces identifiants dans Kubernetes
+
+```shell
+kubectl create secret generic regcred \
+    --from-file=.dockerconfigjson=<path/to/.docker/config.json> \
+    --type=kubernetes.io/dockerconfigjson
+```
+
+Si vous avez besoin de plus de contrôle (par exemple, pour définir un Namespace ou une étiquette sur le nouveau secret), vous pouvez alors personnaliser le secret avant de le stocker.
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:72]:
```suggestion
Si vous avez besoin de plus de contrôle (par exemple, pour définir un Namespace ou un label sur le nouveau secret), vous pouvez alors personnaliser le secret avant de le stocker.
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 9]
```diff
@@ -0,0 +1,209 @@
...
+```
+
+{{< note >}}
+Si vous utilisez le credentials store de Docker, vous ne verrez pas cette entrée `auth` mais une entrée `credsStore` avec le nom du Store comme valeur. 
+{{< /note >}}
+
+## Créez un Secret basé sur les identifiants existants du Docker {#registry-secret-existing-credentials}
+
+Le cluster Kubernetes utilise le type Secret de `docker-registry` pour s'authentifier avec
+un registre de conteneurs pour en tirer une image privée.
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:62]:
```suggestion
un registre de conteneurs pour y récupérer une image privée.
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 10]
```diff
@@ -0,0 +1,209 @@
+---
+title: Extraction d'une image d'un registre privé
+content_template: templates/task
+weight: 100
+---
+
+{{% capture overview %}}
+
+Cette page montre comment créer un Pod qui utilise un Secret pour extraire une image d'un
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:9]:
```suggestion
Cette page montre comment créer un Pod qui utilise un Secret pour récupérer une image d'un
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 11]
```diff
@@ -0,0 +1,209 @@
...
+dépôt ou registre privé de Docker.
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:10]:
```suggestion
registre privé.
```

File: content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md [hunk 12]
```diff
@@ -0,0 +1,209 @@
...
+* Pour faire cet exercice, vous avez besoin d'un
+[Docker ID](https://docs.docker.com/docker-id/) et un mot de passe.
+
+{{% /capture %}}
+
+{{% capture steps %}}
+
+## Connectez-vous à Docker
+
+Sur votre ordinateur portable, vous devez vous authentifier à un registre afin de récupérer une image privée :
```
Comment (remyleone) [inline content/fr/docs/tasks/configure-pod-container/pull-image-private-registry.md:27]:
```suggestion
Sur votre ordinateur, vous devez vous authentifier à un registre afin de récupérer une image privée :
```

**Pull request merged**